blocks/hero/hero.js

```diff
@@ -0,0 +1,16 @@
+export default function decorate(block) {
+  const pictureEl = block.querySelector('picture');
+  const headingEl = block.querySelector('h1, h2, h3, h4, h5, h6');
+
+  const newDiv = document.createElement('div');
+  newDiv.className = block.className;
+  Array.from(block.attributes).forEach((attr) => {
+    if (!['class'].includes(attr.name)) newDiv.setAttribute(attr.name, attr.value);
+  });
+
+  const textDiv = document.createElement('div');
+  newDiv.append(pictureEl, textDiv);
+  textDiv.append(headingEl);
+
+  block.replaceWith(newDiv);
```

Comment:
Rather than replace the block itself, why not just use replaceChildren? Then you don't neeed to worry about copying the classname or attributes over to the new block.

Reply:
Done.